client/extensions/woocommerce/app/settings/payments/payments-off-site.js

```diff
@@ -7,15 +7,37 @@ import { localize } from 'i18n-calypso';
 /**
  * Internal dependencies
  */
-import Card from 'components/card';
+
 import ExtendedHeader from '../../../components/extended-header';
+import List from '../../../components/list/list';
+import ListHeader from '../../../components/list/list-header';
+import ListItemField from '../../../components/list/list-item-field';
+import PaymentMethodItem from './payment-method-item';
 
 class SettingsPaymentsOffSite extends Component {
 
+	state = {
+		methods: [
+			{
+				label: 'PayPal Standard',
+				isSuggested: true,
+				fee: '2.9% + 30c per transaction',
```

Comment:
This is mock data, there is follow up PR.  Also that data does not come over the API.